xc-admin/packages/xc-admin-frontend/types/index.ts

```diff
@@ -0,0 +1,45 @@
+import { PriceComponent, PriceType, Product } from '@pythnetwork/client'
+import { PublicKey } from '@solana/web3.js'
+import { PriceStatus } from '../utils/PriceStatus'
+
+export interface PythPriceData {
```

Comment:
Aren't these types already in @pythnetwork/client?

Reply:
they are different tho